turbo/app/snapshots_cmd.go

```diff
@@ -797,18 +798,26 @@ func doRetireCommand(cliCtx *cli.Context) error {
 	}); err != nil {
 		return err
 	}
-
+	existBlocksToPrune := true
 	for j := 0; j < 10_000; j++ { // prune happens by small steps, so need many runs
-		if err := db.UpdateNosync(ctx, func(tx kv.RwTx) error {
-			if err := br.PruneAncientBlocks(tx, 100); err != nil {
+		if !existBlocksToPrune {
```

Comment:
can replace by `for existBlocksToPrune {` ?

Reply:
> can replace by `for existBlocksToPrune {` ?

for j := 0; existBlocksToPrune || j < 10_000; j++ { replaced by this or you want smth like
for j := 0; existBlocksToPrune; j++ { ?